static/app/views/alerts/rules/metric/details/sidebar.tsx

```diff
@@ -458,3 +458,8 @@ const TriggerText = styled('span')`
   width: 100%;
   font-weight: ${p => p.theme.fontWeightNormal};
 `;
+
+const StyledFeatureBadge = styled(FeatureBadge)`
+  margin-left: ${space(0.25)};
+  padding-bottom: 3px;
```

Comment:
ideally this would also be a `space` value, maybe we just keep it consistent with `0.25`?

```suggestion
  padding-bottom: ${space(0.25)};
```